recipes-kernel/linux/linux-hostmobility-mainline-4.19/mx4-t30/0001-Add-device-tree-files-for-t30.patch

```diff
@@ -700,14 +857,28 @@ index 00000000..97be156
 +				nvidia,tristate = <TEGRA_PIN_DISABLE>;
 +			};
 +			
-+			/*CANXCVR FRM pull up in the future to disable chip select*/
++			/* SSP-CAN-FRM qqq PULL UP in the future to disable chip select*/
 +			spi2_cs0_px3 {
 +				nvidia,pins = "spi2_cs0_n_px3";
-+				nvidia,function = "gmi";
++				nvidia,function = "spi2";
 +				nvidia,pull = <TEGRA_PIN_PULL_DOWN>;
 +				nvidia,tristate = <TEGRA_PIN_DISABLE>;
 +				nvidia,enable-input = <TEGRA_PIN_DISABLE>;
 +			};
++			spi2_cs2_pw3 { /* USBC-OC in */
++				nvidia,pins = "spi2_cs2_n_pw3";
++				nvidia,function = "spi2";
++				nvidia,pull = <TEGRA_PIN_PULL_NONE>;
++				nvidia,tristate = <TEGRA_PIN_DISABLE>;
++				nvidia,enable-input = <TEGRA_PIN_ENABLE>;
++			};
++			spi2_cs1_n_pw2 { /* MODEM-WAKEUP qqq PULL_UP?*/
```

Comment:
Configured with at commands and if it is active the output from modem is active low (if it is the same as ring0)

PLS8-E_HID_v04.004b , 3.12.3, Remote Wakeup